### ISSUE TITLE
Bug: Hostname is stripped when creating /_template from New Page modal

### ISSUE THREAD

**Issue by rsp9u** [weseek/growi#1005]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|GROWI     |3.4.0|

### Client

| item     | version |
| ---      | --- |
|OS        |Chrome|
|browser   |74.0.3729.169|

How to reproduce? (再現手順)
---------------------------

1. Access to Demo page (https://demo.growi.org/)
1. Create template page for / from New Page modal

What happens? (症状)
---------------------

- Moved to `https://_template/#edit-form`

What is the expected result? (期待される動作)
-------------------------------------------

- Move to `https://demo.growi.org/_template/#edit-form`

Note
----

- On other pages, Edit template button works.

**Comment by rsp9u** [comment]
When I checked this modal HTML with Devtools, the HTML was as follows;

```html
<a id="link-to-template" href="//__template#edit-form" class="fcbtn btn btn-outline btn-rounded btn-primary btn-1b">
  <i class="icon-fw icon-doc"></i>
  <span id="create-template-button-link">編集</span>
</a>
```

Then, I read the client code and found the cause.

https://github.com/weseek/growi/blob/229ed8c2c5a3a5d510af010ce831cd0a10e1bc18/src/server/views/modal/create_page.html#L75

---

It should be switched by endsWith("/") in the same way as below;

https://github.com/weseek/growi/blob/2f09d0308f3355484e37dcd0e66f45b33db9b44c/src/server/views/modal/create_template.html#L21

Or, it may be better to refer to `$('#template-button-children').attr('href')` and `$('#template-button-decendants').attr('href')`.

**Comment by shinoka7** [comment]
@rsp9u Thanks for letting us know and figuring out the problem. 
Considering what you mentioned, I think the problem was because of the extra `/` at the end of
`{{ page.path || path }}`, where getting rid of it seemed to solve it for me.

**Comment by rsp9u** [comment]
@shinoka7 Thanks for your response!
So, I'm afraid that your solution wouldn't work well in the other than root page.
For example, this variable in `https://demo.growi.org/seqdiag-test` is like below;
```
// modify href
const value = $(this).val();
const pageName = (value === 'children') ? '_template' : '__template';
const link = '/seqdiag-test/' + pageName + '#edit-form';
$('#link-to-template').attr('href', link);
```
If you get rid of `/` from `/seqdiag-test/`, the link uri would be `/seqdiag-test_template#edit-form`.
As mentioned above, it would be better to switch while the template rendering.

**Comment by shinoka7** [comment]
@rsp9u Ah, you're right! I was only considering the two cases...
As of now, we will probably go with filtering the path, depending on whether or not it has a trailing slash.

**Comment by rsp9u** [comment]
@shinoka7 I've looked at the conversations in PR, certainly that's better than my idea.
Thanks for the explanation!

**Comment by yuki-takei** [comment]
fixed by #1008

**Comment by yuki-takei** [comment]
fixed by #1021
